web/static/js/keyboard.js

```
@@ -528,6 +532,42 @@
+    // register: add programmatic shortcut entries for shortcuts handled by
+    // inline scripts that are not discoverable via data-sw-shortcut attributes.
+    // Entries survive rebuild() calls; use unregister(scope) to remove them.
+    // Each entry: { key, label } -- scope and kind:'manual' are set here.
+    register: function (scope, entries) {
+      if (!Array.isArray(entries)) {
+        if (window.console && console.error) {
+          console.error('[swKbd] register: entries must be an array');
+        }
+        return;
+      }
+      for (var i = 0; i < entries.length; i++) {
```

Biome (2.5.0)

[error] This var should be declared at the root of the enclosing function.

(lint/correctness/noInnerDeclarations)